src/IL/OpCodeExtensions.cs

```diff
@@ -47,6 +47,11 @@ public static OpCode ReadOpCode(this Func<byte> reader)
             return op != 0xfe ? OneByteOpCodes[op] : TwoBytesOpCodes[reader()];
         }
 
+        public static OpCode ReadOpCode(this MethodBodyReader.ByteBuffer buffer) {
```

Comment:
You need to clarify the _intent_ behind the two `ReadOpCode` overloads. Why do we need two overloads? What is the catch? When should one used over the other?